app/app.go

```diff
@@ -193,6 +196,7 @@ var (
 		crisis.AppModuleBasic{},
 		slashing.AppModuleBasic{},
 		ibc.AppModuleBasic{},
+		ibctm.AppModuleBasic{},
```

Comment:
https://github.com/cosmos/ibc-go/pull/2825/files, this just registers the codec's as it's not in ibc core anymore